modules/core/src/main/java/org/apache/ignite/internal/mem/unsafe/UnsafeMemoryProvider.java

```diff
@@ -41,6 +41,9 @@
     /** */
     private IgniteLogger log;
 
+    /** */
```

Comment:
I suggest to add javadoc here describing pupose of this field. We now understand it, but after time new community member may require to find out which init is saved in this flag